app/services/schema_service.rb

```diff
@@ -9,7 +9,7 @@ def self.all_schemas
   def self.find_schema_by_name(name)
     GovukSchemas::Schema.find(publisher_schema: name)
   rescue Errno::ENOENT
-    message = "Could not find publisher schema with name: #{name}"
-    raise CommandError.new(code: 404, message:)
+    msg = "Could not find publisher schema with name: #{name}"
+    raise CommandError.new(code: 404, message: msg)
```

Comment:
@pezholio curious about this one! Is this a thing?!

Reply:
It's a pattern I've seen elsewhere in the app and seems to work well

Reply:
Sorry, missed the context from the commit. This is indeed a thing, introduced in Ruby 3.1 - https://dev.to/baweaver/ruby-3-1-shorthand-hash-syntax-first-impressions-19op

Reply:
Oh nice, I somehow couldn't find that in my search. 

> Posted on 12 Sept 2021

Slightly late to this party! I suppose it's okay once you know but otherwise not in love with it. As it's used elsewhere I'll drop this commit.